src/views/pages/address/choose-address-form.njk

```diff
@@ -22,25 +22,33 @@
         name: "selected_address",
         value: cache | dump,
         fieldset: {
-        legend: {
+          legend: {
             text: pageTitle,
             isPageHeading: true,
             classes: "govuk-fieldset__legend--xl"
-        }
+          }
+        },
+        hint: {
+          text: pageTitleHint
         },
         items: addressItems
     }) }}
 
+    <p>
+      <a href="{{ props.currentUrl | replace(props.pageType, props.data.enterManualAddressPageType) }}" class="govuk-body-m govuk-link" data-event-id="address-not-on-list">{{ addressLink }}</a>
+    </p>
```

Comment:
moved the manual address link above continue button for all choose address pages, some other ages like confirm address have it in the same location except for postcode pages which i assume will have to move it retrospectively as well.

Reply:
Sounds ok. Assume it needs to be in the same place on all the 'choose address' screens. Perhaps worth adding a comment for testers pointing this out, or is it obvious?

Reply:
i can add a comment